src/views/clusteroverview/MonitoringTab/migrations-card/components/BandwidthConsumptionChart/BandwidthConsumptionChart.tsx

```diff
@@ -87,6 +93,7 @@ const BandwidthConsumptionChart: React.FC<BandwidthConsumptionChartProps> = ({ d
           scale={{ x: 'time', y: 'linear' }}
           containerComponent={
             <ChartVoronoiContainer
+              labelComponent={<ChartTooltip style={{ fontSize: 8 }} />}
```

Comment:
suggestion:
make a const for fontSize like so: 
`const fontSize = { fontSize:8 };`
and use it where needed :)

Reply:
thank you :)